crates/turborepo/build.rs

```diff
@@ -1,77 +1,13 @@
-use std::{
-    env,
-    path::{Path, PathBuf},
-    process::Command,
-};
+use std::{env, fs, path::PathBuf, process::Command};
 
 fn main() {
-    let is_ci_release = matches!(env::var("PROFILE"), Ok(profile) if profile == "release")
-        && env::var("RELEASE_TURBO_CLI")
-            .map(|val| val == "true")
-            .unwrap_or(false);
-    let lib_search_path = if is_ci_release {
-        expect_release_lib()
-    } else {
-        build_debug_libturbo()
-    };
-    println!(
-        "cargo:rerun-if-changed={}",
-        lib_search_path.to_string_lossy()
-    );
-    println!(
-        "cargo:rustc-link-search={}",
-        lib_search_path.to_string_lossy()
-    );
-    println!("cargo:rustc-link-lib=turbo");
-
-    let target = build_target::target().unwrap();
-    let bindings = bindgen::Builder::default()
-        .header(lib_search_path.join("libturbo.h").to_string_lossy())
-        // Tell cargo to invalidate the built crate whenever any of the
-        // included header files changed.
-        .parse_callbacks(Box::new(bindgen::CargoCallbacks))
-        .allowlist_function("nativeRunWithArgs")
-        .allowlist_type("GoString")
-        .generate()
-        .expect("Unable to generate bindings");
-
-    let out_dir = env::var("OUT_DIR").unwrap();
-    let out_path = Path::new(&out_dir).join("bindings.rs");
-
-    bindings
-        .write_to_file(out_path)
-        .expect("Couldn't write bindings!");
-
-    if target.os == build_target::Os::MacOs {
-        println!("cargo:rustc-link-lib=framework=cocoa");
-        println!("cargo:rustc-link-lib=framework=security");
+    let is_ci_release = matches!(env::var("PROFILE"), Ok(profile) if profile == "release");
```

Comment:
This means running `cargo run --release` will break, is that intentional?

Reply:
Running `cargo run --release` won't break, it just doesn't build the Go binary. Our release process builds the Go binary separately, setting a bunch of values for each supported platform.

Reply:
Well won't break, but it also won't be able to run

Reply:
Correct, if you want to run a release build of the rust binary, you are responsible for providing the Go binary outside of the rust build process. 

For development work, the debug build still calls over to the `Makefile` to build the Go binary for you.

Reply:
We could keep the `RELEASE_TURBO_CLI` variable to only not build the Go binary if we're running in CI